packages/agent-ops/frontend/src/app/deployWizard/DeployWizardSelectField.scss

```
@@ -0,0 +1,4 @@
+.deploy-agent-wizard-dropdown {
+  --deploy-agent-wizard-dropdown-max-width: 28rem;
+  width: min(100%, var(--deploy-agent-wizard-dropdown-max-width));
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)